Formula/odin.rb

```diff
@@ -0,0 +1,36 @@
+class Odin < Formula
+  desc "The Odin Programming Language"
+  homepage "https://odin-lang.org/"
+  url "https://github.com/odin-lang/Odin/archive/v0.12.0.tar.gz"
+  sha256 "8356124c8cc7e08ac39872e5bb10593a412e67f81df621124097facd9b2b26cc"
+
+  depends_on "llvm"
```

Comment:
Does it not work with macOS llvm?

Reply:
It works but it also needs other binaries from llvm toolchain not present in default installation like `opt`.